pkgs/by-name/py/pywebcopy/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchPypi,
+  python3,
+  python3Packages,
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "pywebcopy";
+  version = "7.0.2";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-H10GiPyAvX6UVM5by4TqW+z6tcwPqskMnTML3BWJdVU=";
+  };
+
+  postPatch = ''
+    touch requirements.txt
+  '';
+
+  nativeBuildInputs = with python3Packages; [ setuptools ];
```

```suggestion
  build-system = with python3Packages; [ setuptools ];
```

this is a new preferred way for python packages
https://nixos.org/manual/nixpkgs/unstable/#python